profiler/src/ProfilerEngine/Datadog.Profiler.Native/LiveObjectInfo.cpp

```diff
@@ -10,7 +10,6 @@ LiveObjectInfo::LiveObjectInfo(std::shared_ptr<Sample> sample, uintptr_t address
     :
     _address(address),
     _weakHandle(nullptr),
-    _timestamp(timestamp),
```

Comment:
Hum... If we don't need the timestamp, it means that we don't compute the lifetime of the live objects.
Maybe the implementation is not complete and we need to keep it for now

Reply:
Why not adding when we want to complete the implementation ?
For now it's useless